### ISSUE TITLE
Unit training quantity doesn't get refreshed upon confirm

### ISSUE THREAD

**Issue by quanlinc** [bspiritxp/ixa-helper#8]
The remaining quantity is supposed to be refreshed after one submit, but it's not until a second click on confirm button.

Might have something to do with promise chain timing, left a comment in code, needs to be addressed in future, low priority I think.

**Comment by quanlinc** [comment]
Fixed in this commit 
https://github.com/bspiritxp/ixa-helper/pull/7/commits/78a3e9aa251c02af79dee32d5ca8faeaef90208b
